src/ui/settings.tsx

```diff
@@ -0,0 +1,112 @@
+import * as IRF from 'internet-roadtrip-framework';
+import { render } from 'solid-js/web';
+
+import styles, { stylesheet as moduleCss } from '../style.module.css';
+import { getVehicle, setVehicle } from '../pano/vehicle';
+import { getNoiseActive, setNoiseActive, getNoiseVolume, setNoiseVolume } from '../noise/noise';
+import { getWheelRotation, setWheelRotation } from '../pano/wheel';
+import { getOptionsRotation, setOptionsRotation } from '../pano/options';
+import { getMaxZoom, setMaxZoom } from '../pano/scene';
+
+if (IRF.isInternetRoadtrip) {
+	const { container: settings } = IRF.ui.panel.createTabFor(
+		{ ...GM.info, script: { name: 'LOtW' } },
+		{ tabName: 'LOtW', style: moduleCss },
+	);
+
+	function SettingsPanel() {
+		return (
+			<div>
+				<div class={styles['settings-item']}>
+					<span class={styles['setting']}>Vehicle</span>
+					<hr />
+					<div class={styles['setting']} on:mousedown={(e) => e.stopPropagation()}>
+						<input
+							type='checkbox'
+							class={IRF.ui.panel.styles.toggle}
+							checked={getVehicle()}
+							onClick={() => setVehicle(!getVehicle())}
+						/>
+					</div>
+				</div>
+				<div class={styles['settings-item']}>
+					<span class={styles['setting']}>Rotate Wheel w/ Camera</span>
+					<hr />
+					<div class={styles['setting']} on:mousedown={(e) => e.stopPropagation()}>
+						<input
+							type='checkbox'
+							class={IRF.ui.panel.styles.toggle}
+							checked={getWheelRotation()}
+							onClick={() => setWheelRotation(!getWheelRotation())}
+						/>
+					</div>
+				</div>
+				<div class={styles['settings-item']}>
+					<span class={styles['setting']}>Rotate Options w/ Camera</span>
+					<hr />
+					<div class={styles['setting']} on:mousedown={(e) => e.stopPropagation()}>
+						<input
+							type='checkbox'
+							class={IRF.ui.panel.styles.toggle}
+							checked={getOptionsRotation()}
+							onClick={() => setOptionsRotation(!getOptionsRotation())}
+						/>
+					</div>
+				</div>
+				<div class={styles['settings-item']}>
+					<span class={styles['setting']}>Ambient Noise</span>
+					<hr />
+					<div class={styles['setting']} on:mousedown={(e) => e.stopPropagation()}>
+						<input
+							type='checkbox'
+							class={IRF.ui.panel.styles.toggle}
+							checked={getNoiseActive()}
+							onClick={() => setNoiseActive(!getNoiseActive())}
+						/>
+					</div>
+				</div>
+				<div class={styles['settings-item']}>
+					<span class={styles['setting']}>Noise Volume</span>
+					<div class={styles['setting']} on:mousedown={(e) => e.stopPropagation()}>
+						<input
+							type='range'
+							class={IRF.ui.panel.styles.slider}
+							min='0'
+							max='100'
+							step='1'
```

Comment:
Personal preference of mine

```suggestion
							min={0}
							max={100}
							step={1}
```

Reply:
I'm definitely willing to go back on this though. Its a very small detail.

I already got my W with the double quotes for JSX.

Reply:
No, no. You have a point with this.